pkg/cpi/mockingUtils.go

```diff
@@ -273,20 +266,18 @@ func GetMockResponseByTestTypeAndMockFunctionName(mockFuntionName, testType stri
 
 	case "IntegrationDesigntimeArtifactUpdateMockResponse":
 		if response == nil && error == nil {
-
-			res := http.Response{
-				StatusCode: 400,
-				Body: ioutil.NopCloser(bytes.NewReader([]byte(`{
-							"code": "Bad Request",
-							"message": {
-							"@lang": "en",
-							"#text": "invalid request"
-							}
-						}`))),
-			}
-			return &res, errors.New("Unable to get status of integration artifact, Response Status code:400")
+			return NegtiveResForIntegrationDesigntimeArtifactUpdateMockResponse()
 		}
 	case "GetIntegrationDesigntimeArtifactMockResponse":
+		if response == nil && error == nil {
```

Comment:
this `if` is in each case, could be done for the entire `switch`

Reply:
done